src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx

```diff
@@ -4,108 +4,2980 @@ tags:
 
-In many cases SNMP offers a lot of data, but much of that data does not provide actionable information. Or, the data provides value that is so uncommon and low impact, it might not be worth bringing into your New Relic account.
+## What kinds of OIDs should I include? [#oids]
 
-You want to focus on collecting data that lets you know if there is anything that would stop the device from being able to perform whatever functions you expect it to perform. Building on from that, you should collect measurements that tell you how well it is performing those functions.
+In many cases SNMP offers a lot of data, but little provides actionable information. The [sample SNMP walk](#snmp-polling) above has 2,022 rows of results from OIDs, and the [resulting profile](https://github.com/kentik/snmp-profiles/blob/main/profiles/kentik_snmp/apc/apc_ups.yml) collects about 50 total OIDs. Following the basic patterns of **"Golden Signals"** defined by the [Site Reliability Engineering (SRE) discipline](https://sre.google/sre-book/monitoring-distributed-systems/); focus on collecting data that lets you know if there is anything that would stop the device from being able to perform whatever functions you expect it to perform. Building on from that, you should collect measurements that tell you how well it is performing those functions.
 
-**Example 1:** For a device operating as a VPN concentrator, we would collect high-level system metrics like:
+**Example 1:**
+
+For a device operating as a VPN concentrator, we would collect high-level system metrics like:
 
 * CPU and memory utilization
 * Hardware sensor information to make sure that the device isn't going to shut itself down due to things like fan failures
 * OIDs that tell us about the aggregated connections and throughput
 
-**Example 2:** An example of data that is available but provides very poor value is an OID table that lists all the running processes on a network appliance. Coming from a server admin perspective, that might sound useful, but since this is an appliance, you normally do not have the capability or the need to do anything with the processes that run inside it. Polling and storing tables with hundreds of items that you can't actually do anything with would not be efficient.
+**Example 2:**
+
+An example of data that is available but provides very poor value is an OID table that lists all the running processes on a network appliance. That might sound useful, but since this is an appliance, you normally do not have the capability or the need to do anything with the processes that run inside it. Polling and storing tables with hundreds of items that you can't actually do anything with would not be efficient.
```

Comment:
```suggestion
An OID table that lists all the running processes on a network appliance is an example of data that is available but not helpful. 

Collecting all running processes might sound useful, but since this is an appliance, you normally do not have the capability or the need to do anything with the processes that run inside it. Polling and storing tables with hundreds of items that you can't actually do anything with would not be efficient.
```